templates/land-see/partials/nav.snip.html

```diff
@@ -0,0 +1,55 @@
+{{!
+<!---
+Copyright 2016 The AMP Start Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+}}
+
+{{#navbar}}
```

Comment:
Again same feedback as above - we should try and use the components code as much as possible

> I looked into using the ampstart navbar component some more and realized that {{#fixeditems}} could be used for the search icon. The problem I ran into there is that I'd like to use the font-awesome icon instead of passing in an svg. So I'm continuing to use my custom partial, but updated the "search" icon to be more customizable via JSON variables.

You are free to modify the components library to fit in anything that does not already have a place for it in there.

Also make sure the stuff you use has an appropriate distribution license , Apache-2 or CC0 I guess for images. Not sure if font awesome is re-distributable

Reply:
Eliminating font-awesome in next commit. That will solve a couple issues. I'll now just pass in our svg to existing `{{#fixeditems}}`, and we won't need to rely on any font-awesome dependencies.